src/components/UI/organisms/UserRegistrationForm.js

```diff
@@ -1,20 +1,47 @@
 import {Form, Button, Row, Col} from 'react-bootstrap';
+import React from 'react';
+
 const UserRegistrationForm = (props) => {
+	const [name, setName] = React.useState('');
+	const [password, setPass] = React.useState('');
+	const [username, setEmail] = React.useState('');
```

Comment:
make sure when using `useState` you have a getter and a setter for the same prop in format `[<name>, set<Name>]`, having different names will make it really confusing when using these fields, I recommend changing to something like
```suggestion
	const [username, setUsername] = React.useState('');
```